eBCSgen/API/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .ebcsgen_ctl_model_checking import ctl_model_checking
+from .ebcsgen_generate_ts import generate_ts
+from .ebcsgen_pctl_model_checking import pctl_model_checking
+from .ebcsgen_pctl_parameter_synthesis import pctl_parameter_synthesis
+from .ebcsgen_sbml_export import sbml_export
+from .ebcsgen_simulate import simulate
+from .ebcsgen_static_analysis import static_analysis
```

Comment:
```suggestion
```

I don't think these imports are necessary. Is there a specific reason for including them?